packer/scripts/debian/networking.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/bash -eux
 
 rm /etc/udev/rules.d/70-persistent-net.rules
-mkdir /etc/udev/rules.d/70-persistent-net.rules
```

Comment:
Why are we removing this ? Also, do we not build locally any other version of debian which might be affected by this ?